lib/src/app.dart

```diff
@@ -10,13 +10,13 @@ class MyApp extends StatefulWidget {
 }
 
 class _MyAppState extends State<MyApp> {
-  final _blocs = <Bloc>[];
+  final _blocs = <DiceBloc>[];
 
   @override
   void initState() {
     super.initState();
-    DiceBloc bloc = DiceBloc();
-    _blocs.add(bloc);
+    _blocs.add(new DiceBloc());
```

Comment:
We shouyld be creating more than one instance of a Bloc, the logic for both dices should be handled inside the DiceBloc

Reply:
Done.